.werft/values.nodeAffinities_0.yaml

```diff
@@ -0,0 +1,62 @@
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: gitpod.io/service_0
+          operator: Exists
+
+components:
+
+  wsDaemon:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: gitpod.io/workspace_0
+              operator: Exists
+
+  registryFacade:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: gitpod.io/workspace_0
+              operator: Exists
+
+
+  workspace:
+    affinity:
+      default: gitpod.io/workspace_0
+
+# Sub-Charts
+
+rabbitmq:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: gitpod.io/service_0
```

Comment:
The `service` nodepool is meant for dev-services only, not any preview services. It should be "workspace"/"meta" for all of them.